fib-prime.js

```diff
@@ -1,34 +1,35 @@
-var ispnum = function(num) {
-  for(var i = 2; i < num; i++)
-    if  (num % i === 0) return false;
-  return num > 1;
-};
+const isPrime = num => {
+  for (let i = 2, s = Math.sqrt(num); i <= s; i++)
+    if (num % i === 0) return false
 
-const fibonacci = (num) => {
-  if (num <= 1)return 1;
-  return fibonacci(num - 1) + fibonacci(num - 2);
-};
+  return num > 1
+}
+
+const fibonacci = num => {
+  if (num <= 1) return 1
+  return fibonacci(num - 1) + fibonacci(num - 2)
+}
+
+const nextPrimeFibonacci = input => {
```

Comment:
Changed function and argument names for better readable.
Used ES6 Function Expression to prevent reassignment function.